Add documentation for the Jellyfin media player (#23910)
home-assistant/home-assistant.github.io: source/_integrations/jellyfin.markdown

The Jellyfin integration exposes a [Jellyfin](https://jellyfin.org/) server as a Media Source in Home Assistant.
Support is currently limited to music and movie libraries only. Other libraries will not appear in the Media Browser. This integration has been tested with Jellyfin server version 10.6.4 and later.

> Additionally this integration sets up every media session connected to the Jellyfin
> server as media player in Home Assistant.
> This provides media controls for the sessions from Home Assistant.
> Browsing media inside Home Assistant in a player's context provides all libraries
> of type Movie and Series.

Reviewer: ```suggestion
Additionally, this integration sets up every media session connected to the Jellyfin
server as a media player in Home Assistant to provide media controls for each session.

Browsing media inside Home Assistant in a player's context provides all libraries
of type Movie and Series.
```
